packages/nimble-components/build/generate-workers/tests/render-worker.spec.ts

```diff
@@ -0,0 +1,9 @@
+import { HealthStatus } from '../source/health-status';
+import { RenderWorker } from '../source/render-worker';
+
+describe('RenderWorker', () => {
```

Comment:
I am also not sure if these tests are run with the CI, can you check and see if you need to make a new test script?

Reply:
you are right, the test was not running, now it should be fine